testsuite/scripts/tutorials/test_electrodes_1.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2019-2022 The ESPResSo project
+#
+# This file is part of ESPResSo.
+#
+# ESPResSo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ESPResSo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import unittest as ut
+import importlib_wrapper
+import numpy as np
+
+tutorial, skipIfMissingFeatures = importlib_wrapper.configure_and_import(
+    "@TUTORIALS_DIR@/electrodes/electrodes_part1.py",
+    script_suffix="@TEST_SUFFIX@")
+
+
+@skipIfMissingFeatures
+class Tutorial(ut.TestCase):
+
+    def test_force(self):
+        msg = 'The force for particle 1 should agree with the analytical expression.'
+        np.testing.assert_allclose(tutorial.elc_forces_axial[:, 0],
+                                   tutorial.analytic_force_centered(tutorial.r / tutorial.BJERRUM_LENGTH, tutorial.box_l_z), rtol=1, err_msg=msg)
```

Comment:
rtol=1 is way too much

Reply:
Adressed with [c30845c](https://github.com/espressomd/espresso/pull/4784/commits/c30845c5f2b6ec0e88b65146f11b7df3a31c0446)